components/Frame/SecondaryNav.js

```diff
@@ -0,0 +1,158 @@
+  SUBHEADER_HEIGHT_MOBILE,
+  ZINDEX_HEADER
+} from '../constants'
+
+export const SecondaryNav = ({
+  secondaryNav,
+  dark,
+  showSecondary,
+  router,
+  hasOverviewNav,
+  t
+}) => {
+  const active = matchPath(router.asPath)
+  console.log(secondaryNav)
+  return (
+    <>
+      {hasOverviewNav ? (
+        <div
+          {...styles.miniNav}
+          style={{
+            backgroundColor: dark ? colors.negative.primaryBg : '#fff'
+          }}
+        >
+          <div {...styles.navContainer}>
+            <NavLink
+              style={{ color: dark ? colors.negative.text : colors.text }}
+              route='index'
+              active={active}
+              minifeed={true}
+            >
+              {t('navbar/front')}
+            </NavLink>
+            <NavLink
+              style={{ color: dark ? colors.negative.text : colors.text }}
+              prefetch
+              route='feed'
+              active={active}
+              minifeed={true}
+            >
+              {t('navbar/feed')}
+            </NavLink>
+            <NavLink
+              style={{ color: dark ? colors.negative.text : colors.text }}
+              route='discussion'
+              active={active}
+              hoverColor={colors.primary}
+              minifeed={true}
+            >
+              {t('navbar/discussion')}
+            </NavLink>
+            <Sections dark={dark} active={active} minifeed={true} />
+          </div>
+        </div>
+      ) : (
+        secondaryNav && (
+          <div
+            {...styles.secondaryNav}
+            style={{
+              opacity: showSecondary ? 1 : 0,
+              transition: 'opacity 0.2s ease-out',
+              backgroundColor: dark ? colors.negative.primaryBg : '#fff'
+            }}
+          >
+            {secondaryNav}
+          </div>
+        )
+      )}
+    </>
+  )
+}
+
+const styles = {
+  secondaryNav: css({
+    position: 'absolute',
+    zIndex: ZINDEX_HEADER,
+    top: HEADER_HEIGHT_MOBILE,
+    left: 0,
+    right: 0,
+    height: SUBHEADER_HEIGHT_MOBILE,
+    display: 'flex',
+    justifyContent: 'flex-start',
+    borderBottom: `1px solid ${colors.divider}`,
+    padding: `0px ${Math.floor((HEADER_HEIGHT_MOBILE - 26) / 2)}px`,
+    [mediaQueries.mUp]: {
+      top: HEADER_HEIGHT,
+      justifyContent: 'center',
+      padding: `0px ${Math.floor((HEADER_HEIGHT - 26) / 2)}px`
+    }
+  }),
+  miniNav: css({
+    position: 'absolute',
+    overflowY: 'hidden',
+    zIndex: ZINDEX_HEADER,
+    top: HEADER_HEIGHT_MOBILE,
+    height: SUBHEADER_HEIGHT_MOBILE,
+    left: 0,
+    right: 0,
+    display: 'flex',
+    justifyContent: 'flex-start',
+    borderBottom: `1px solid ${colors.divider}`,
+    [mediaQueries.mUp]: {
+      top: HEADER_HEIGHT,
+      justifyContent: 'center'
+    },
+    '& a': {
+      display: 'block',
+      whiteSpace: 'nowrap',
+      fontSize: 14,
+      margin: '0 16px',
+      ':active': {
+        fontWeight: 'bold'
+      },
```

Comment:
You should never manually set `fontWeight` or just `fontFamily` but always use `fontStyles` which contains a valid combinations of those two. In this case you can directly assign:
```suggestion
      ':active': fontStyles.sansSerifMedium,
```

(If you had additional styles you could spread it: `{ ...fontStyles.sansSerifMedium, color: 'red' }`)

Reply:
jo hab ich sonst immer gemacht, weiss nicht warum hier nicht XD